_includes/glossary_links.adoc

```diff
@@ -0,0 +1,6 @@
+// glossary_links.adoc
+// The attributes defined provide links to anchors on the glossary page.  Please also remember to add the
+// corresponding definition to glossary.adoc.
+:glossary-businessstakeholder-: link:/glossary#glossary-businessstakeholder[Business Stakeholder]
+:glossary-customersupport: link:/glossary#glossary-customersupport[Customer Support]
+:glossary-sre: link:/glossary#glossary-sre[SRE]
```

Comment:
```suggestion
:glossary-sre: pass:[<abbr title="Site Reliability Engineer">link:/glossary#glossary-sre[SRE]</abbr>]
```

^Just a thought: this would give hover text to the abbreviation. It's still linked to the glossary for people to click-through for more info.

Reply:
I did wonder about using that technique so that you could hover over any glossary term and the hover text would show the expansion.    To so that I would need a second attribute containing the glossary item's description which would be referenced on pages/glossary.adoc and to give the hover text.  I decided that was over-kill.

I think you are suggesting a special case for SRE as it is an abbreviation.